### PR TITLE
CMFGEN version check workflow

### PR DESCRIPTION
**Pull request by s-rathi**
### :pencil: CMFGEN version check workflow without the CMFGEN atomic data

**Type:** :rocket: `feature`

### :vertical_traffic_light: Testing

How did you test these changes?

- [ ] Testing pipeline
- [X] Other method (describe)
- [ ] My changes can't be tested (explain why)

### :ballot_box_with_check: Checklist

- [X] I requested two reviewers for this pull request
- [ ] I updated the documentation according to my changes
- [ ] I built the documentation by applying the `build_docs` label

> **Note:** If you are not allowed to perform any of these actions, ping (@) a contributor.

### CODE REVIEW

File: .github/workflows/cmfgen_data_check.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+name: Check CMFGEN data update workflow
+
+on:
+  schedule:
+    - cron: '0 0 * * 5'  #runs on every Friday midnight
+  workflow_dispatch:
+  
+env:
+  SOURCE_URL: https://kookaburra.phyast.pitt.edu/hillier/cmfgen_files/atomic_data_15nov16.tar.gz
```
Comment (atharva-2001) [inline .github/workflows/cmfgen_data_check.yml:9]: Is there a new URL?

File: .github/workflows/cmfgen_data_check.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
```
Comment (atharva-2001) [inline .github/workflows/cmfgen_data_check.yml:14]: perhaps we can move this above?

File: .github/workflows/cmfgen_data_check.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        id: compare
+        run: |
+          git add .
+          git diff --quiet --exit-code --cached
+          FILE_CHANGE=$?
+          echo "change=$FILE_CHANGE" >> "$GITHUB_OUTPUT"
+
+      - name: Commit files to git
+        id: commit_changes
+        if: ${{ steps.compare.outputs.change == 1 }}
```
Comment (atharva-2001) [inline .github/workflows/cmfgen_data_check.yml:50]: This is good. Another job indicating the exit code status might be helpful.

**Review by atharva-2001 — COMMENTED** [review]
This looks very good. A few small comments. Also, a new dataset has been released, there should be two versions of the atomic data in the repo, right?

File: .github/workflows/cmfgen_data_check.yml (hunk 1 above)
Comment (andrewfullard) [reply .github/workflows/cmfgen_data_check.yml:9]: There is, but we should run this first, then change it, then see what has changed between atomic data versions IMO

**Review by andrewfullard — APPROVED** [review]

**Review by atharva-2001 — APPROVED** [review]

**Pull request merged**